Live-Snap/Live-Snap/Networking/IsValidSnapchatUsernameEndpointRequest.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  IsValidSnapchatUsernameEndpointRequest.swift
+//  Live-Snap
+//
+//  Created by Baby on 3/13/18.
+//  Copyright © 2018 Oleg Abalonski. All rights reserved.
+//
+
+import UIKit
+import Alamofire
+
+class IsValidSnapchatUsernameEndpointRequest {
+    
```

Comment:
Unfortunately this approach won't work to check if a snapchat username is valid. We can still use this class to check if a bitmoji is available for the username. Possibly display a toggle box if the bitmoji is available?